fundamentals/01_datastructures.ipynb

```diff
@@ -75,7 +75,7 @@
     "multi-dimensional arrays while `Dataset` combines multiple arrays.\n",
     "\n",
     "Both classes are most commonly created by reading data.\n",
-    "To learn how to create a DataArray or Dataset manually, see the **Working with labeled data** tutorial.\n",
+    "To learn how to create a DataArray or Dataset manually, see the [Creating Data Structures](01.1_creating_data_structures.html) tutorial.\n",
```

Comment:
```suggestion
    "To learn how to create a DataArray or Dataset manually, see the [Creating Data Structures](01.1_creating_data_structures.ipynb) tutorial.\n",
```

(need to reference the source file rather than built html https://jupyterbook.org/en/stable/content/references.html#reference-content-files)